examples/src/main/scala/org/apache/spark/examples/graphx/LiveJournalPageRank.scala

```diff
@@ -30,14 +25,14 @@ object LiveJournalPageRank {
   def main(args: Array[String]) {
     if (args.length < 1) {
       System.err.println(
-        "Usage: LiveJournalPageRank <edge_list_file>\n" +
+        "Usage: LiveJournalPageRank <edge_list_file> --numEPart=<num_edge_partitions>\n" +
```

Comment:
`--numEPart` is still duplicated. It's added to this line but already shown directly below.